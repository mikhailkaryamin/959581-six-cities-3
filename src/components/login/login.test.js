import React from 'react';
import renderer from "react-test-renderer";
import Login from './login.jsx';

const currentCity = `Paris`;

test(`Render correctly sign in`, () => {
  const tree = renderer
  .create(
      <Login
        currentCity={currentCity}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
