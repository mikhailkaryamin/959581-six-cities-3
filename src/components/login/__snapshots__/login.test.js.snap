// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly sign in 1`] = `null`;
